scripts/coinweight.lua

```diff
@@ -57,10 +59,12 @@ local function createCoinsItem(nodeChar)
 end
 
 ---	This function looks for the "Coins" inventory item if it already exists.
+---	Also matches "Coins (Coins Weight Extension)" for more context in name.
 local function findCoinsItem(nodeChar)
 	for _,nodeItem in pairs(DB.getChildren(nodeChar, 'inventorylist')) do
 		local sItemName = DB.getValue(nodeItem, 'name', '')
-		if sItemName == 'Coins' then
+		if sItemName == COINS_INVENTORY_ITEM_NAME
+		   or string.match(sItemName:lower(), '^%W*coins%W+coins%W+weight%W+extension%W*$') then
```

Comment:
Optional regex matching of the four words 'coins', 'coins', 'weight', and 'extension' in that order, with any amount of whitespace or non-alphanum delimiters.

Reply:
I think this is much too broad and likely to cause incompatibilities with other extensions.

Reply:
However I don't understand match perfectly, so it could be that I am interpreting it wrong.
Is "Coins" required to be in the item name?

Reply:
Yes, all four words must be there.  Maybe just an optional exact match of 'Coins (Coins Weight Extension)' would be sufficient and best.  This is something I can live with on a fork if it's not worth having in the codebase.

Reply:
Oh if all 4 must be there I'm happy :)